### PR TITLE
RESTWS-566: Applying the DrugResource changes from platform 1.12 correctly

### PR DESCRIPTION
**Pull request by mozzy11**
Ticket : https://issues.openmrs.org/browse/RESTWS-566

## Issue I worked on
I  applied the DrugResource changes from platform 1.12 correctly to run when running platform 1.12
.

### CODE REVIEW

**Comment by dkayiwa** [comment]
Did you see the travis build failure?

**Comment by mozzy11** [comment]
oh yes its wat am working on to solve

**Comment by dkayiwa** [comment]
How about DrugResource2_0?

**Comment by mozzy11** [comment]
oops , i thought its only this https://github.com/openmrs/openmrs-module-webservices.rest/blob/master/omod-2.0/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_0/DrugOrderSubclassHandler2_0.java    to be transferd back to 1.12

**Comment by mozzy11** [comment]
Alryt let me also move the drugResource also

**Comment by mozzy11** [comment]
Hello @dkayiwa , i updated the Pr , and also moved DrugResource2_0 back to 1_12

File: omod-1.12/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_12/RestConstants1_12.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_12;
+
+/**
+ * {@link org.openmrs.module.webservices.rest.web.RestConstants} for 1.11 resources.
```
Comment (dkayiwa) [inline omod-1.12/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_12/RestConstants1_12.java]: For which resources?

Comment (mozzy11) [reply omod-1.12/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_12/RestConstants1_12.java]: ops , for 1.12 resources. let me correct that

Comment (mozzy11) [reply omod-1.12/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_12/RestConstants1_12.java]: i have corected it

**Pull request merged**